engine/src/test/groovy/graphql/nadel/e2e/NadelE2EDynamicServiceResolutionTest.groovy

```diff
@@ -0,0 +1,314 @@
+package graphql.nadel.e2e
+
+import graphql.*
+import graphql.execution.AbortExecutionException
+import graphql.execution.ExecutionId
+import graphql.execution.ExecutionIdProvider
+import graphql.execution.ExecutionStepInfo
+import graphql.execution.instrumentation.InstrumentationContext
+import graphql.execution.instrumentation.InstrumentationState
+import graphql.execution.instrumentation.SimpleInstrumentationContext
+import graphql.language.SourceLocation
+import graphql.nadel.*
```

Comment:
can you change your formatter to never use * imports? Have a look at the GraphQL formatter in the GJ repo.